cg/cli/set/cases.py

```diff
@@ -67,18 +65,18 @@ def families(
 
     LOG.info("Would alter cases:")
 
-    for case in cases:
-        LOG.info(case)
+    for case_obj in cases:
+        LOG.info(case_obj)
 
     if not (click.confirm(CONFIRM)):
         raise click.Abort
 
-    for case in cases:
+    for case_obj in cases:
```

Comment:
Or is it to not shadow the function below? If so I suggest renaming to `cases_to_alter` and `case_to_alter`

Reply:
Yes, the problem her is the shadowing but then I will use your suggestion :)